lib/xenops_server.ml

```diff
@@ -1556,6 +1565,9 @@ and trigger_cleanup_after_failure_atom op t =
   | PCI_unplug id ->
     immediate_operation dbg (fst id) (PCI_check_state id)
 
+  (* If VGPU_set_active fails, we will not do any cleanup action *)
```

Comment:
Why not just put it together with `VGPU_start`?

Reply:
Oh, yes. I didn't notise that we have VGPU_start in master which does not exist before. 
Thanks. Will update them together.